tests/constants/test-entities/connections.ts

```diff
@@ -0,0 +1,7 @@
+export enum ConnectionsUrls {
+    Connection = '/connections/dr0y3yyi1gjk2-connection',
```

Comment:
I suggest to give more more meaningful names for entries. Because there can be more than one connection and it would be nice to determine them by name

Reply:
+1